tests/test_rpms.py

```diff
@@ -137,6 +134,8 @@ def git_subject(git_dir, ref):
     cmd = ['git', 'show', '-s', '--format=%s', ref]
     proc = Popen(cmd, cwd=git_dir, stdout=PIPE)
     out, _ = proc.communicate()
+    if not isinstance(out, str) and isinstance(out, bytes):
```

Comment:
Similarly here, I don't think you should need to add this yourself as Popen can decode output from the process just fine.  universal_newlines=True likely does what you want.

Reply:
Fixed.